app/javascript/controllers/auth_restriction_controller.js

```diff
@@ -31,26 +31,30 @@ export default class extends Controller {
     this.stanfordRestrictionDismissButtonTarget.hidden = false
   }
 
-  dismissStanfordRestriction() {
-    this.stanfordRestrictionTarget.hidden = true
+  hideAuthRestrictionMessages() {
+    this.authRestrictionMessagesContainerTarget.hidden = true
```

Comment:
Could this be `this.element.hide = true`?  I don't think we need a special target for the container itself.

Reply:
fixed